dev/sparktestsupport/modules.py

```diff
@@ -1501,6 +1501,18 @@ def __hash__(self):
     ],
 )
 
+pyspark_pipelines = Module(
+    name="pyspark-pipelines",
+    dependencies=[],
```

Comment:
Let's add dependencies same as `pyspark_sql`. If you have a Spark Connect tests, it has to be `pyspark_connect`.

Reply:
👍